linux+kernel+rpi64+lts: update to 6.12.34

chenx97 opened the pull request:
Topic Description
-----------------

- linux+kernel+rpi64: new, 6.18.6
- linux-kernel-rpi64: new, 6.18.6+rpi20260120
- linux+kernel+rpi64+lts: update to 6.12.34
    ... and forward version number to defines
- linux-kernel-rpi64-lts: update to 6.12.34+stable20250702
    - use the stable tag of the rpi tree
    - assert that the .config file stays unmodified after olddefconfig
    - minimize uses of eval
    - forward the base version instead of $PKGVER to inner scripts
    - cap mouse polling to 500Hz on RPi3
- rpi-firmware-boot: update to 1.20250915

Package(s) Affected
-------------------

- linux+kernel+rpi64: 6.18.6
- linux+kernel+rpi64+lts: 6.12.34
- linux-kernel-rpi64-${__VER}: 6.18.6+rpi20260120
- linux-kernel-rpi64-lts-${__VER}: 6.12.34+stable20250702
- rpi-firmware-boot: 1.20250915

Security Update?
----------------

No

Build Order
-----------

```
#buildit rpi-firmware-boot linux-kernel-rpi64-lts linux+kernel+rpi64+lts linux-kernel-rpi64 linux+kernel+rpi64
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AArch64 `arm64`

Cyanoxygen commented:
<img width="2560" height="1361" alt="image" src="https://github.com/user-attachments/assets/1365cce5-5251-42af-a355-b45bd553a390" />
LGTM on raspberry pi 5.

Cyanoxygen commented:
LGTM on Raspberry Pi 4B.

<img width="2560" height="1390" alt="image" src="https://github.com/user-attachments/assets/45ee8cbc-ca2a-4673-b243-2c7ec9186587" />

Cyanoxygen commented:
…… And for the next time I think we can introduce my fork of Petitboot to this platform, to allow it to choose the kernel variant.

chenx97 committed:
linux-kernel-rpi64: new, 6.18.6+rpi20260120

chenx97 committed:
rpi-firmware-boot: update to 1.20260408

chenx97 committed:
linux+kernel+rpi64+lts: update to 6.12.47

chenx97 committed:
linux+kernel+rpi64: new, 6.18.6

chenx97 committed:
linux-kernel-rpi64-lts: update to 6.12.47+stable20250916

- use the stable tag of the rpi tree
- assert that the .config file stays unmodified after olddefconfig
  - sort configs like other kernel packages
- minimize uses of eval
- forward the base version instead of $PKGVER to inner scripts
- cap mouse polling to 500Hz on RPi3
